docs/src/running-tests-csharp.md

```diff
@@ -67,8 +67,30 @@ You can run a single test, a set of tests or all tests. Tests can be run on diff
 
 For more information see [selective unit tests](https://docs.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests?pivots=mstest) in the Microsoft docs.
 
+## Debugging Tests
+
+Since Playwright runs in Node.js, you can debug it with your debugger of choice. Playwright comes with the Playwright Inspector which allows you to step through Playwright API calls, see their debug logs and explore [selectors](./selectors.md).
+
+```bash tab=bash-bash lang=csharp
+PWDEBUG=1 dotnet test
+```
+
+```batch tab=bash-batch lang=csharp
+set PWDEBUG=1
+dotnet test
+```
+
+```powershell tab=bash-powershell lang=csharp
+$env:PWDEBUG=1
+dotnet test
+```
+
+<img width="712" alt="Playwright Inspector" src="https://user-images.githubusercontent.com/883973/108614092-8c478a80-73ac-11eb-9597-67dfce110e00.png"></img>
+
+Check out our [debugging guide](./debug.md) to learn more about the [Playwright Inspector](./debug.md#playwright-inspector) as well as debugging with [Browser Developer tools](./debug.md#browser-developer-tools).
+
+
 ## What's Next
 
-- [Debug tests with the Playwright Debugger](./debug.md)
```

Comment:
lets bring that back?

Reply:
its already showing the debugging basics so its not really whats next as you want them to go to codegen and traceviewer next